corehq/apps/users/models.py

```diff
@@ -2902,6 +2907,12 @@ class HQApiKey(models.Model):
     ip_allowlist = ArrayField(models.GenericIPAddressField(), default=list)
     domain = models.CharField(max_length=255, blank=True, default='')
     role_id = models.CharField(max_length=40, blank=True, default='')
+    is_active = models.BooleanField(default=True)
```

Comment:
I notice this has `default=True`, just mentioning that this will require a full table update during the migration. I don't expect it to be an issue with this table since it's small and low traffic but thought I'd mention it anyway.